services/voting-node/pyproject.toml

```diff
@@ -12,7 +12,7 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
 # formatting
 black = "^23.1.0"
```

Comment:
`black`, `ruff` should be "development" dependencies because we don't need to deploy them.
See: https://python-poetry.org/docs/managing-dependencies#dependency-groups

Reply:
Done.